### PR TITLE
repair usa case counts

### PR DESCRIPTION
**Pull request by hamishgibbs**
NYT changed to cumulative cases. I have removed cumsum and behaviour should be same as previous version of the function.

### CODE REVIEW

**Comment by hamishgibbs** [comment]
I believe build failing because of slow url for `get_interventions_data`

File: R/get_spain_regional_cases.R [hunk 1]
```diff
@@ -83,10 +83,10 @@ get_spain_regional_cases <- function(dataset = "cases_provincial"){
       dplyr::group_by(province) %>%
       dplyr::arrange(date, .by_group = T) %>%
       dplyr::mutate(cases_daily = cases_cum - lag(cases_cum, default = dplyr::first(cases_cum)),
-                    hospital_daily = hospital_cum - lag(hospital_cum, default = first(hospital_cum)),
-                    icu_daily = icu_cum - lag(icu_cum, default = first(icu_cum)),
-                    deaths_daily = deaths_cum - lag(deaths_cum, default = first(deaths_cum)),
-                    recover_daily = recover_cum - lag(recover_cum, default = first(recover_cum))) %>% 
+                    hospital_daily = hospital_cum - lag(hospital_cum, default = dplyr::first(hospital_cum)),
```
Comment (seabbs) [inline R/get_spain_regional_cases.R:86]: Need to add `dplyr::lag` into the imports

File: R/get_us_regional_cases.R [hunk 2]
```diff
@@ -44,7 +44,14 @@ get_us_regional_cases <- function(level = 'state', out = 'timeseries'){
   
   get_nyt_state <- function(path, out){
     
-    cases <- mem_read(path)
+    cases <- mem_read(path) %>%
```
Comment (seabbs) [inline R/get_us_regional_cases.R:47]: Is this the same as cases - lag(cases)?

Comment (seabbs) [reply R/get_us_regional_cases.R:47]: Do you think it is a good idea to have cumulative as an argument?

File: tests/testthat/custom_tests/expect_colname.R [hunk 3]
```diff
@@ -0,0 +1,3 @@
+expect_colname <- function(expected_colname, colnames){
```
Comment (seabbs) [inline tests/testthat/custom_tests/expect_colname.R:1]: NIce

File: tests/testthat/test-get_belgium_regional_cases.R [hunk 4]
```diff
@@ -28,3 +28,30 @@ test_that("get_belgium_regional_cases cases works as expected", {
   
 })
 
+
+test_that("get_belgium_regional_cases data sources are unchanged", {
```
Comment (seabbs) [inline tests/testthat/test-get_belgium_regional_cases.R:32]: Nice :)

**Review by seabbs — CHANGES_REQUESTED** [review]
Looks good. Just need to clarify a few things. Also seeing a build failure....

**Pull request closed without merging**